scripts/enable-backend.sh

```
@@ -57,26 +53,27 @@ set -- "${positionalArgs[@]+"${positionalArgs[@]}"}"
 
 if [ -n "$1" ]; then
-  instance="$1"
+  instanceArg="$1"
 else
-  echo "What is the name of the instance?"
-  read -r instance
+  echo "Which instance? (name, or path to the instance directory, e.g. '.')"
+  read -r instanceArg
 fi
+resolveInstancePath "$instanceArg" || exit 1
+instance=$(getVar "$path/.env" INSTANCE_NAME)
```

Shellcheck (0.11.0)

[warning] path is referenced but not assigned.

(SC2154)